app/jsx/pages/PageBase.jsx

```diff
@@ -205,6 +205,7 @@ class PageBase extends React.Component
       if (/^nanocad/.test(unit_id)) { this.runExtGATracker('nanocadTracker', 'UA-17962781-1') }
       if (/^uciem_westjem/.test(unit_id)) { this.runExtGATracker('westjemTracker', 'UA-34762732-1') }
       if (/^ucla_epss/.test(unit_id)) { this.runExtGATracker('epssTracker', 'UA-111990925-2') }
+      if (/^refract/.test(unit_id)) { this.runExtGATracker('epssTracker', 'UA-130336975-1') }      
```

Comment:
The tracker name should be unique I believe. I.e. instead of 'epssTracker' it should be 'refractTracker'.